es6-promises/index.js

```diff
@@ -0,0 +1,11 @@
+const takeAChance = require('./take-a-chance');
+
+const promiseNick = takeAChance('Nick');
+
+promiseNick.catch(error => {
+  console.error(error);
+});
+
+promiseNick.then(value => {
+  console.log(value);
+});
```

Comment:
This will work but traditionally we chain these together, and do the success first. For example:
```JAVASCRIPT
promiseNick.then(value => {
  console.log(value);
}).catch(error => {
  console.error(error);
});
```